pimd/pim_msg.h

```diff
@@ -158,10 +158,18 @@ struct pim_encoded_source_ipv6 {
 typedef struct pim_encoded_ipv4_unicast pim_encoded_unicast;
 typedef struct pim_encoded_group_ipv4   pim_encoded_group;
 typedef struct pim_encoded_source_ipv4  pim_encoded_source;
+typedef struct ip 			ipv_hdr;
```

Comment:
It is not right to copy paste the changes from other's PRs. If you want to take the changes take the commits and mention properly. Few changes are copied from #10707.

Reply:
#10707 has pim_br.c changes of this pull-request.

Reply:
No, It does not have the changes from this pull request. I have not taken any change from your PR. It is the same change because it is converting in_addr to pim_addr.